refinedstorage2-platform-forge/src/test/java/com/refinedmods/refinedstorage2/platform/common/importer/ImporterTest.java

```diff
@@ -0,0 +1,228 @@
+
+            // Act
+            final ItemStack damagedDiamondChestplate = DIAMOND_CHESTPLATE.getDefaultInstance();
+            damagedDiamondChestplate.setDamageValue(500);
+            prepareChest(helper, pos.east(), Items.DIRT.getDefaultInstance(),
+                    DIAMOND_CHESTPLATE.getDefaultInstance(), damagedDiamondChestplate);
+
+            importer.setFuzzyMode(false);
+            importer.setFilters(Set.of(asResource(DIAMOND_CHESTPLATE.getDefaultInstance())));
+            importer.setFilterMode(FilterMode.BLOCK);
+
+            // Assert
+            sequence
+                    .thenWaitUntil(() -> assertContainerContainsExactly(helper, pos.east(),
+                            DIAMOND_CHESTPLATE.getDefaultInstance()))
+                    .thenWaitUntil(storageContainsExactly(
+                            helper,
+                            pos,
+                            new ResourceAmount(asResource(DIRT), 11),
+                            new ResourceAmount(asResource(STONE), 15),
+                            new ResourceAmount(asResource(damagedDiamondChestplate), 1)
+                    ))
+                    .thenSucceed();
+        });
+    }
+
+    @GameTest(template = "empty_15x15")
+    public static void shouldImportItemFuzzyBlocklist(final GameTestHelper helper) {
+        preparePlot(helper, Direction.EAST, (importer, pos, sequence) -> {
+            // Arrange
+            sequence.thenWaitUntil(networkIsAvailable(helper, pos, network -> {
+                insert(helper, network, DIRT, 10);
+                insert(helper, network, STONE, 15);
+            }));
+
+            // Act
+            final ItemStack damagedDiamondChestplate = DIAMOND_CHESTPLATE.getDefaultInstance();
+            damagedDiamondChestplate.setDamageValue(500);
+            prepareChest(helper, pos.east(), Items.DIRT.getDefaultInstance(),
+                    DIAMOND_CHESTPLATE.getDefaultInstance(), damagedDiamondChestplate);
+
+            importer.setFuzzyMode(true);
+            importer.setFilters(Set.of(asResource(DIAMOND_CHESTPLATE.getDefaultInstance())));
+            importer.setFilterMode(FilterMode.BLOCK);
+
+            // Assert
+            sequence
+                    .thenWaitUntil(() -> assertContainerContainsExactly(helper, pos.east(),
+                            DIAMOND_CHESTPLATE.getDefaultInstance()))
+                    .thenWaitUntil(() -> assertContainerContainsExactly(helper, pos.east(), damagedDiamondChestplate))
+                    .thenWaitUntil(storageContainsExactly(
+                            helper,
+                            pos,
+                            new ResourceAmount(asResource(DIRT), 11),
+                            new ResourceAmount(asResource(STONE), 15)
+                    ))
+                    .thenSucceed();
+        });
+    }
+
+    @GameTest(template = "empty_15x15")
+    public static void shouldImportItemNotFuzzyAllowlist(final GameTestHelper helper) {
```

Comment:
"Not fuzzy" is the default, I think? So in that case:
```suggestion
    public static void shouldImportItemAllowlist(final GameTestHelper helper) {
```
Makes the test goal more understandable (negations are hard)

Reply:
same remark for the other tests